GVFS/GVFS/CommandLine/GVFSVerb.cs

```diff
@@ -140,6 +147,8 @@ public static bool TrySetRequiredGitConfigSettings(Enlistment enlistment)
                 { "merge.renames", "false" },
                 { "pack.useBitmaps", "false" },
                 { "pack.useSparse", "true" },
+                { "pack.depth", "0" },
+                { "pack.window", "0" },

```

Comment:
If we do this change we can remove the additions I did in GitProcess.cs
$"pack-objects {packFilePath} --non-empty **--window=0 --depth=0** -q",

Reply:
We can. There just wasn't a way to do it (currently) through the `multi-pack-index repack` command. It would require wiring up a lot of complexity that isn't worth it.

Reply:
**ACTUALLY...** this also shuts down delta calculation during `git push`. That's _not_ something we should do, as the server doesn't redeltify. We rely on clients sending decent deltas or the total space will blow up.

I'm going to see if running the process with `-c pack.depth=0 -c pack.window=0` carries those config settings through the `run_command()` method.

Reply:
(I noticed this because the `git pack-objects` command in the perf suite sped up.)

Reply:
Nice Catch!